client/src/api/apiUtils.js

```diff
@@ -0,0 +1,14 @@
+const superagent = require('superagent');
+require('dotenv').config();
+
+export const getAds = setAds => {
```

Comment:
doesn't make sense to have a get and set in the same definition. ideally, you want getAds() to return the data to your component. then your component stores that return in a response variable and pass into setAds(response)